pkg/svcat/service-catalog/sdk.go

```diff
@@ -17,12 +17,68 @@ limitations under the License.
 package servicecatalog
 
 import (
+	"time"
+
+	apiv1beta1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
 	svcatclient "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
 	"github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset/typed/servicecatalog/v1beta1"
+	apicorev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/version"
 	"k8s.io/client-go/kubernetes"
 	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
 )
 
+// SvcatAPI is an interface containing the variuos actions in the svcat pkg lib
+// This interface is then faked with Counterfeiter for the cmd/svcat unit tests
+type SvcatAPI interface {
```

Comment:
Can we come up with a different name for this that doesn't include "API"? The reason I went with SDK on the wrapper that I created was that it really wasn't a blind api client like the regenerated go client already is.